src/styles.css

```diff
@@ -94,7 +94,7 @@
 
 .IBMChat-messages button[disabled="true"] {
   cursor: default;
-  opacity: .7;
+  opacity: .5;
```

Comment:
can you try that out with other color themes and make sure .5 opacity works for a chat widget with a white background? IIRC we spent forever going back and forth on the opacity levels.